File: nj2k/testData/newJ2k/issues/kt-19550.java
public class TestToStringReturnsNullable {
    public static class Base {
        public String string;
    }

    public static class Ctor extends Base {
        public Ctor(String string) {
            this.string = string;
        }
    }

    public static class Derived extends Ctor {
        public Derived(String string) {
            super(string);
        }

        @Override
        public String toString() {
            return string;
        }
    }
}

File: nj2k/testData/newJ2k/issues/kt-19550.kt
class TestToStringReturnsNullable {
    open class Base {
        var string: String? = null
    }

    open class Ctor(string: String) : Base() {
        init {
            this.string = string
        }
    }

    class Derived(string: String) : Ctor(string) {
        override fun toString(): String {
            return string!!
        }
    }
}